tools/ARIAtools/sequential_stitching.py

```diff
@@ -984,6 +985,10 @@ def write_GUNW_array(output_filename: Union[str, Path],
 
     array_type = gdal_array.NumericTypeCodeToGDALTypeCode(array.dtype)
 
+    # File must be physically extracted, cannot proceed with VRT format.
```

Comment:
if we leave this line here, that the function lose it modularity...we could just easily remove the option of selecting the format and set it every as ENVI. 

If this is needed for aria_extract, maybe it would be better to set it there to be  ENVI, aria_extract.py line 1135

Reply:
@sssangha  can you please add the closing figure/axes lines to the plotting function. 

fig.clear()
plt.close(fig)

Although, we should keep the plotting option  FALSE as default, as there still might be some memory leak https://github.com/matplotlib/matplotlib/pull/22002

Reply:
@mgovorcin sounds good, I addressed your comments. I had to clean-up similar `outputformat` checks in other functions, but now this check is handled explicitly in aria_extract

Reply:
looks good